File: python/baskara.py
from math import sqrt

a = float(input("Coeficiente a: "))
b = float(input("Coeficiente b: "))
c = float(input("Coeficiente c: "))

delta = b ** 2 - 4 * a * c

if delta < 0:
    print("Esta equacao nao possui raizes reais")
else:
    x1 = (-b + sqrt(delta)) / (2 * a)
    x2 = (-b - sqrt(delta)) / (2 * a)
    print(f"X1 = {x1:.4f}")
    print(f"X2 = {x2:.4f}")

File: javascript/baskara.js
const a = parseFloat(prompt("Digite o valor de A"));
const b = parseFloat(prompt("Digite o valor de B"));
const c = parseFloat(prompt("Digite o valor de C"));

const delta = b * b - 4 * a * c;

if (delta < 0) {
    console.log("não possui raízes reais");
} else {
    const x1 = (-b + Math.sqrt(delta)) / (2 * a);
    const x2 = (-b - Math.sqrt(delta)) / (2 * a);

    console.log("x1 = %.4f", x1);
    console.log("x2 = %.4f", x2);
}